src/main/src/main/java/org/geoserver/config/SpringResourceAdaptor.java

```diff
@@ -74,22 +86,30 @@ public boolean isOpen() {
 
     @Override
     public URL getURL() throws IOException {
-        return Resources.find(resource).toURI().toURL();
+        return getFile().toURI().toURL();
     }
 
     @Override
     public URI getURI() throws IOException {
-        return Resources.find(resource).toURI();
+        return getFile().toURI();
```

Comment:
Okay I see what you did there; using getFile() to do the null check in a single location